yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -438,27 +438,12 @@ export class PXEService implements PXE {
     });
   }
 
-  public async getTxReceipt(txHash: TxHash): Promise<TxReceipt> {
```

Comment:
Moved the implementation to aztec node since I needed an access to block source to be able to populated the block number and block hash of the receipt.